KesaApplication/Utility.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Utility.swift
+//  KesaApplication
+//
+//  Created by Joon Lee on 2016-05-30.
+//  Copyright © 2016 kesa. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class Utility {
+    
+    /* Function to help check for the email validity */
+    func isValidEmailFormat (testEmail:String) -> Bool {
```

Comment:
Consider making it a static method.  Also please change the argument name from testEmail -> email.
I don't think the comment is required for this function since it is pretty obvious from the method's name. Also, please make the sentence grammatically correct!